### PR TITLE
Adds react-events package for internal testing

### PR DESCRIPTION
**Pull request by trueadm**
This PR adds a `react-events` package that will act as the home for the user-facing APIs for the experimental event API. This package is only intended for internal testing right now. This also adds a bundle to the build script so we can build the `react-events` package to consume internally.

This PR will be a dependency for https://github.com/facebook/react/pull/15112.

### CODE REVIEW

File: packages/react-events/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "react-events",
+  "description": "React is a JavaScript library for building user interfaces.",
+  "keywords": [
+    "react"
+  ],
+  "version": "16.8.4",
```
Comment (gaearon) [inline packages/react-events/package.json]: Do you want to start this at `0.1.0` instead?

File: packages/react-events/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * @flow
+ */
+
+'use strict';
+
+const ReactEvents = require('./src/ReactEvents');
+
+// TODO: decide on the top-level export form.
+// This is hacky but makes it work with both Rollup and Jest.
+module.exports = ReactEvents.default || ReactEvents;
```
Comment (acdlite) [inline packages/react-events/index.js]: I don't think we've been doing this on our new packages. See: https://github.com/facebook/react/blob/master/packages/react-cache/index.js

Comment (gaearon) [reply packages/react-events/index.js]: Ah good catch

Comment (trueadm) [reply packages/react-events/index.js]: Indeed! https://github.com/facebook/react/pull/15152

**Pull request merged**